app/src/main/java/com/lastaoutdoor/lasta/ui/screen/social/components/AddFriendDialog.kt

```diff
@@ -62,43 +70,80 @@ fun AddFriendDialog(
                     LocalContext.current.getString(R.string.fr_email_add),
                     modifier = Modifier.testTag("SubHeader"),
                     textAlign = TextAlign.Center)
-                AddFriendForm(friendRequestFeedback, requestFriend)
+                AddFriendForm(
+                    friendRequestFeedback, friendSuggestions, requestFriend, fetchFriendSuggestions)
               }
         }
       }
 }
 
 @Composable
-private fun AddFriendForm(friendRequestFeedback: String, requestFriend: (String) -> Unit) {
+private fun AddFriendForm(
+    friendRequestFeedback: String,
+    friendSuggestions: List<UserModel>,
+    requestFriend: (String) -> Unit,
+    fetchFriendSuggestions: (String) -> Unit
+) {
 
   // to hide the keyboard
   val focusManager = LocalFocusManager.current
   // text inside the textfield
   var text by remember { mutableStateOf("") }
 
-  TextField(
-      value = text,
-      onValueChange = { text = it },
-      label = { Text(LocalContext.current.getString(R.string.email)) },
-      modifier = Modifier.fillMaxWidth().padding(8.dp).testTag("EmailTextField"),
-      keyboardOptions = KeyboardOptions(imeAction = ImeAction.Done),
-      keyboardActions =
-          KeyboardActions(
-              onDone = {
-                focusManager.clearFocus()
-                requestFriend(text)
-              }))
+  Column(modifier = Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
+    TextField(
+        value = text,
+        onValueChange = { newValue -> text = newValue },
+        label = { Text(LocalContext.current.getString(R.string.email)) },

```

Comment:
If we now search by username, it doesn't make a lot of sense to write email in the textfield

Reply:
I mean it still works with email addresses and it's a good thing that it does because the check is done via email, so I think it is relevant to leave this there

Reply:
Yes but how can the user know that it is possible to just enter names ? (of course he can try, but the goal of an indication in a textfield is to tell what is expected in the field) 

Would it be okay to write Email OR user name ?